docs/ops/sequence/AUGRUSequence_10.md

```diff
@@ -0,0 +1,145 @@
+# AUGRUSequence  {#openvino_docs_ops_sequence_AUGRUSequence_10}
+
+**Versioned name**: *AUGRUSequence-10*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUSequence* operation represents a series of AUGRU cells (GRU with attentional update gate).
+
+**Detailed description**: The difference between *AUGRUSequence* and [GRUSequence](./GRUSequence_5.md) is the additional attention score input `A`, which is a multiplier for the update gate.
+The formula was described in the [paper arXiv:1809.03672](https://arxiv.org/abs/1809.03672).
+
+The sequence can be connected differently depending on `direction` attribute that specifies the direction of traversing of input data along sequence dimension or specifies whether it should be a bidirectional sequence.
+
+The most of the attributes are in sync with the specification <a href="https://github.com/onnx/onnx/blob/master/docs/Operators.md#gru">ONNX GRU</a> operator.
+
+
+**Attributes**
+
+* *hidden_size*
+
+  * **Description**: *hidden_size* specifies hidden state size.
+  * **Range of values**: a positive integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+* *activations*
+
+  * **Description**: *activations* specifies activation functions for gates, there are two gates, so two activation functions should be specified as a value for this attributes
+  * **Range of values**: any combination of *relu*, *sigmoid*, *tanh*
+  * **Type**: a list of strings
+  * **Default value**: *sigmoid,tanh*
+  * **Required**: *no*
+
+* *activations_alpha, activations_beta*
+
+  * **Description**: *activations_alpha, activations_beta* attributes of functions; applicability and meaning of these attributes depends on chosen activation functions
+  * **Range of values**: a list of floating-point numbers
+  * **Type**: `float[]`
+  * **Default value**: None
+  * **Required**: *no*
+
+* *clip*
+
+  * **Description**: *clip* specifies bound values *[-C, C]* for tensor clipping. Clipping is performed before activations.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Default value**: *infinity* that means that the clipping is not applied
+  * **Required**: *no*
+
+* *direction*
+
+  * **Description**: Specify if the RNN is forward, reverse, or bidirectional. If it is one of *forward* or *reverse* then `num_directions = 1`, if it is *bidirectional*, then `num_directions = 2`. This `num_directions` value specifies input/output shape requirements.
+  * **Range of values**: *forward*, *reverse*, *bidirectional*
+  * **Type**: `string`
+  * **Required**: *yes*
+
+* *linear_before_reset*
+
+  * **Description**: *linear_before_reset* flag denotes, if the output of hidden gate is multiplied by the reset gate before or after linear transformation. It behaves according to the *ONNX GRU* described in the [ONNX documentation](https://github.com/onnx/onnx/blob/master/docs/Operators.md#GRU).
```

Comment:
Let us also elaborate this instead of referring ONNX documentation please.

Reply:
Included in the math formula